src/common/utxobased/keymanager/keymanager.ts

```diff
@@ -923,7 +923,7 @@ export async function makeTx(args: MakeTxArgs): Promise<MakeTxReturn> {
   const utxoPicker = coin.utxoPicker ?? utxopicker.utxoPicker
 
   const utxopicking =
-    args.forceUseUtxo != null ?? false
+    args.forceUseUtxo.length > 0
```

Comment:
You can always combine both tests into this: `(args.forceUseUtxo?.length ?? 0) > 0`

Reply:
Beautiful!

Reply:
Wait, why do we need this? `forceUseUtxo` is never `undefined`.